napari/_qt/containers/layers/qt_layer_list.py

```diff
@@ -0,0 +1,44 @@
+from typing import TYPE_CHECKING
+
+from qtpy.QtGui import QFont
+from qtpy.QtWidgets import QStyleOptionViewItem, QWidget
+
+from ....layers import Layer
+from ....utils import config
+from ...containers.list import QtListView
+from ._layer_model import QtLayerListModel
+from .delegate import LayerDelegate
+
+if TYPE_CHECKING:
+    from ....components.layerlist import LayerList
+
+
+class QtLayerList(QtListView[Layer]):
+    _list: 'LayerList'
+    model_class = QtLayerListModel
+
+    def __init__(self, root: 'LayerList', parent: QWidget = None):
+        super().__init__(root, parent)
+        self.setItemDelegate(LayerDelegate())
+        fnt = QFont()
+        fnt.setPixelSize(12)
+        self.setFont(fnt)
+
+        if config.async_loading:
```

Comment:
I'm guessing this is left over from before, seems a little out of place. I can clean up in another PR or later ... just making a note for now

Reply:
yeah, that's the one thing that was on QtLayerList that I didn't quite know what to do with, so I just moved it over.